MODULE.bazel

```
@@ -538,3 +537,38 @@ use_repo(
     "busybox",
     "busybox_linux_amd64",
 )
+
+bazel_dep(name = "rules_webtesting", version = "0.2.1")
```

```suggestion
bazel_dep(name = "rules_webtesting")
```
Make it obvious that the dep is overridden.